test/TestDummies/DummyLoggerConfigurationExtensions.cs

```diff
@@ -63,13 +63,20 @@ public static LoggerConfiguration WithDummyThreadId(this LoggerEnrichmentConfigu
 
         public static LoggerConfiguration DummyRollingFile(
             this LoggerSinkConfiguration loggerSinkConfiguration,
-            List<string> objectBinding,
+            List<Binding> objectBinding,
```

Comment:
@nblumhardt In fact, a test using this method should have failed because it sets json incorrectly:
```
 var json = @"{
                ""Serilog"": {            
                    ""Using"": [""TestDummies""],
                    ""WriteTo"": [{
                        ""Name"": ""DummyRollingFile"",
                        ""Args"": {""pathFormat"" : ""C:\\"",
                                   ""objectBinding"" : [ { ""foo"" : ""bar"" }, { ""abc"" : ""xyz"" } ] }
                    }]        
                }
            }";
```

`objectBinding` is not a array of string, it is array of objects. I came across this error when adding support for arrays 
 of custom types.